modules/tinymce/src/plugins/table/test/ts/module/test/TableSizingModeCommandUtil.ts

```diff
@@ -57,14 +58,23 @@ const generateTable = (mode: SizingMode, width: number, rows: number, cols: numb
   return `<table border="1" style="border-collapse: collapse;${tableWidth}">${getColumns()}<tbody>${renderedRows}</tbody></table>`;
 };
 
-const sTableSizingModeScenarioTest = (editor, tinyApis: TinyApis, title: string, description: string, withColGroups: boolean, scenario: Scenario) =>
-  Log.stepsAsStep(title, description, [
+const sTableSizingModeScenarioTest = (editor, tinyApis: TinyApis, title: string, description: string, withColGroups: boolean, scenario: Scenario, expectedEventsLength: number = 1) => {
+  let events = [];
+  editor.on('tablemodified', (e: EditorEvent<{}>) => events.push(e));
+  const clearEvents = Step.sync(() => events = []);
+
+  return Log.stepsAsStep(title, description, [
+    clearEvents,
+    Step.sync(() => Assertions.assertEq('Number of events starts at 0', 0, events.length)),
```

Comment:
Is this actually doing anything meaningful? As we've just cleared it above, so this seems pointless?